CHANGELOG.md

```diff
@@ -14,6 +14,45 @@ Changes by Version
 
 ### UI Changes
 
+1.19.0 (2020-08-25)
+------------------
+
+### Backend Changes
+
+#### New Features
+
+* Reload TLS certificates on change ([#2389](https://github.com/jaegertracing/jaeger/pull/2389), [@pavolloffay](https://github.com/pavolloffay))
+* Add --kafka.producer.batch-min-messages collector flag ([#2371](https://github.com/jaegertracing/jaeger/pull/2371), [@prymitive](https://github.com/prymitive))
+* Make UDP socket buffer size configurable ([#2336](https://github.com/jaegertracing/jaeger/pull/2336), [@kbarukhov](https://github.com/kbarukhov))
+* Enable batch and queued retry processors by default ([#2330](https://github.com/jaegertracing/jaeger/pull/2330), [@pavolloffay](https://github.com/pavolloffay))
+* Add trace anonymizer prorotype ([#2328](https://github.com/jaegertracing/jaeger/pull/2328), [@yurishkuro](https://github.com/yurishkuro))
+* Add native OTEL ES exporter ([#2295](https://github.com/jaegertracing/jaeger/pull/2295), [@pavolloffay](https://github.com/pavolloffay))
+* Define busy error type in processor ([#2314](https://github.com/jaegertracing/jaeger/pull/2314), [@pavolloffay](https://github.com/pavolloffay))
+* Use gRPC instead of tchannel in hotrod ([#2307](https://github.com/jaegertracing/jaeger/pull/2307), [@pavolloffay](https://github.com/pavolloffay))
+* TLS support for gRPC Query server ([#2297](https://github.com/jaegertracing/jaeger/pull/2297), [@jan25](https://github.com/jan25))
```

Comment:
We need to double-check this one. The current master does *not* show the option that this one brought. Was it reverted?

Reply:
@jpkrohling any update on this one? Should it be removed from the change log and moved to the next release?

Reply:
Ok, I tracked this down: https://github.com/jaegertracing/jaeger/pull/2337#pullrequestreview-445414839

Basically, the code is there, but the flag isn't being registered. In the end, this is good as this doesn't expose the flag to users, otherwise we'd have to wait for the PR to split ports before doing this release (or revert this PR).

It would probably be worth adding a "known issues" section to the changelog. Like: 
> The pull request #2297 aimed to add TLS support for the gRPC Query server but the flag registration is missing, so that this feature can't be used at the moment. A fix is planned for the next Jaeger version (1.20).

So, we are good to go, IMO.

Reply:
Additional "Known Issues" section added.